### PR TITLE
pad single digit ints when displaying mac address

### PR DESCRIPTION
**Pull request by andymck**

### CODE REVIEW

File: src/cli/miner_cli_info.erl [hunk 1]
```diff
@@ -362,5 +362,10 @@ format_macs_from_interfaces(IFs)->
     [], IFs).
 
 format_hwaddr(HWAddr)->
-    HWAddr0 = [integer_to_list(B,16) || B <- HWAddr ],
+    HWAddr0 = [pad_and_format_int(B) || B <- HWAddr ],
```
Comment (Vagabond) [inline src/cli/miner_cli_info.erl]: Worth looking at `blockchain_utils:bin_to_hex/1` instead?

Comment (Vagabond) [inline src/cli/miner_cli_info.erl]:
```suggestion
   blockchain_utils:bin_to_hex(list_to_binary(HWAddr)).
```

Comment (andymck) [reply src/cli/miner_cli_info.erl]: looks good.  I tested your suggestion locally and does the job

**Comment by Vagabond** [comment]
I think the only annoying thing is that it outputs lowercase hex, maybe check if we want uppercase?

**Comment by andymck** [comment]
will check with redbeard, also need to remove a now spurious list:concat

**Pull request merged**